addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1482,6 +1489,9 @@ const Wysiwyg = Widget.extend({
         return new Promise(function () {});
     },
     _onDocumentMousedown: function (e) {
+        if (!e.target.classList.contains('o_editable_date_field_linked')) {
```

Comment:
@Zinston As the original behavior was:
1. When I click on a date field, add `o_editable_date_field_linked` class on linked date field
2. When I click somewhere, if this is not the same as the previous element that got clicked, I remove all the `o_editable_date_field_linked` class everywhere

But I didn't find a way to get the last clicked element in new implementation so I changed it a bit:
1. When I click on a date field, add `o_editable_date_field_linked` on all field related o this date (self included, which is different from previous implementation)
2. When I click somewhere, if I am not one of the linked date, remove that class.

Not sure why I did not implemented it that way at first, seems easier actually.

Anyway, all of that explanation just because I am not 100% sure if this is the right place to put this line, but it seems to be the only event that trigger when you click everywhere on screen (including right panel which should obviously also remove the class).

Reply:
@rdeodoo
> I didn't find a way to get the last clicked element
There is a `this.lastElement` in `wysiwyg.js`, you could probably have used that.
Anyway this also seems like a good solution.

> it seems to be the only event that trigger when you click everywhere on screen
Just know that it only triggers on `$editable` so not exactly everywhere on the screen... Just everywhere within the editor. In the website builder that happens to be the whole page but that's not the case everywhere. Is that still what you want?

Reply:
I am not too sure how to check if it looks bad, would need to be able to edit a date field in the backend in the editor.
I guess it would keep the box-shadow on linked date that was clicked if the user then click outside the editor.
Doesn't seems that bad but I don't know how to check it

Reply:
I looked around in the views and couldn't find a single date field in the backend in the editor so I guess the problem doesn't really exist. And even if it did, I'm not even sure clicking outside the editor _should_ remove that class (as you're basically putting the editor on pause by acting on something else for a while). So I'd say this is overall acceptable.

Reply:
I don't think that the class being removed or not being removed on click outside is a big deal. I, in fact, am not even sure what would be better.
I'll you judge if you want one or the other, but in case you want a click only on the `$editable` element, I'll need a bit of help to find the correct handler, as that one the only one suiting the need of the frontend edition as mentioned in my first comment here.

> Anyway, all of that explanation just because I am not 100% sure if this is the right place to put this line, but it seems to be the only event that trigger when you click everywhere on screen (including right panel which should obviously also remove the class).